css_tasks/task11/scss/_decoration.scss

```diff
@@ -109,16 +107,67 @@
       content: "";
     }
     &::after {
-      position: absolute;
-      margin-top: 1rem;
-      top: 105%;
-      left: calc(70px + (100 - 70) * ((100vw - 300px) / (1920 - 300)));
+      @include carouselText;
       @include decorativeText;
-      content: "John Doe";
     }
   }
 }
+.carousel-item.active {
+  display: flex;
+}
+.carousel-item.forward {
+  animation: fade;
+  animation-duration: 1.5s;
+}
+.carousel-item.backward {
+  animation: fade-backward;
+  animation-duration: 1.5s;
+}
+.decoration__wrapper--carousel-first
+  .carousel-first
+  .decoration-carousel__text:after,
+.decoration__wrapper--carousel-second
+  .carousel-first
+  .decoration-carousel__text:after {
+  content: "John Doe";
```

Comment:
Try to leave content only in HTML
https://developer.mozilla.org/en-US/docs/Web/CSS/attr#content_property

Reply:
and then you don't need to duplicate this in CSS